### ISSUE TITLE
OSI: Create command script via the repo commands, instead of executing the command online.

### ISSUE THREAD

**Issue by martins-1992** [www-splitcells-net/net.splitcells.network#22]
This would make the solution a lot more portable and easy to debug.
Currently, when something fails, it is hard to understand what is being executed.

In the end the frontend meta repo commands, should create a script consisting of frontend commands for each repo.
These could be executed via a shell or analyzed.
Each line of the script should be independent of each other line.

This would also prevent complex dependencies between each command,
In other words, the aim is to create a very simple rewrite system.
In general, this could be so simple, that bash would be enough for such a system.
